app-frontend/app/src/components/bar/Navigation.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+import {style} from "typestyle";
+import {NavLink} from "react-router-dom";
+
+const {wrapper, bar} = {
+  wrapper: style({
+    display: "block",
+    maxWidth: "1024px",
+    margin: "auto",
+    padding: "20px",
+    $nest: {
+      "a.link": {
+        fontSize: "24px",
+        color: "#F0610B",
+        textDecoration: "none",
+        margin: "10px",
+
+        $nest: {
+          "&.active": {
+            textDecoration: "underline"
+          }
+        }
+      }
+    }
+  }),
+  bar: style({
+    backgroundColor: "rgba(242, 242, 242, 0.7)",
+    $nest: {
+      "& > *": {
+        boxSizing: "border-box"
+      }
+    }
+  })
+};
```

Comment:
Why did the styles go to the `.tsx` file, and not `.less`?

Reply:
Some styles initially was done in less, later project switched to typestyle. It was easier to maintain only required classes, and reference them from components. Also there are was idea to do easy programmatically-defined themes (light, dark)